src/admin-views/widgets/list.php

```diff
@@ -23,6 +23,39 @@
 }
 
 foreach ( $admin_fields as $field_id => $field ) {
+	// Handle a section start. May contain a section title.
+	if ( false !== stripos( $field_id, 'section_start' ) ) {
```

Comment:
Since we're trying to detect if the `$field_id` **starts** with `section_start` I would use `0 === stripos` here.